concrete/src/Page/Page.php

```diff
@@ -262,6 +269,7 @@ protected function populatePage($cInfo, $where, $cvID)
                 $this->cPointerID = $originalRow['cPointerID'];
                 $this->cIsActive = $originalRow['cIsActive'];
                 $this->cPointerOriginalID = $originalRow['cID'];
+                $this->cPointerOriginalSiteTreeID = $row['siteTreeID'];
```

Comment:
Shouldn't this be `$this->cPointerOriginalSiteTreeID = $originalRow['siteTreeID'];` ?

Reply:
Very sharp! I will change that.